config/config.go

```diff
@@ -55,8 +60,8 @@ type (
 		// InternalHttpServer is the config for starting a http.Server
 		// to serve some internal APIs
 		InternalHttpServer HttpServerConfig `yaml:"internalHttpServer"`
-		// ClientAddress is the address for API service to call AsyncService's internal API
-		ClientAddress string `yaml:"clientAddress"`
+		// Membership is the config for cluster members
```

Comment:
Maybe `ClusterConfig` so that it can be more generic to include `numShards` and more in the future